### PR TITLE
Fix/upload

### PR DESCRIPTION
**Pull request by mattzcarey**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

Fixes: #607

## Checklist before requesting a review

Please delete options that are not relevant.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented hard-to-understand areas
- [ ] I have ideally added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: backend/utils/vectors.py [hunk 1]
```diff
@@ -67,38 +67,39 @@ def error_callback(exception):
     print("An exception occurred:", exception)
 
 
-def process_batch(batch_ids):
+def process_batch(batch_ids: List[str]):
```
Comment (mamadoudicko) [inline backend/utils/vectors.py:70]: Sure about the return type ?

Thanks for typing ☺️

**Review by mamadoudicko — DISMISSED** [review]

File: scripts/tables.sql [hunk 2]
```diff
@@ -177,4 +175,4 @@ INSERT INTO migrations (name)
 SELECT '202307111517030_add_subscription_invitations_table'
 WHERE NOT EXISTS (
     SELECT 1 FROM migrations WHERE name = '202307111517030_add_subscription_invitations_table'
-);
\ No newline at end of file
+);
```
Comment (gozineb) [inline scripts/tables.sql:178]: You would also need to change the name of the last migration line 175 and line 177

File: scripts/202307111517031_change_vectors_id_type.sql
Comment (gozineb) [inline scripts/202307111517031_change_vectors_id_type.sql:1]: make it idempotent

File: scripts/tables.sql [hunk 3]
```diff
@@ -177,4 +175,4 @@ INSERT INTO migrations (name)
 SELECT '202307111517030_add_subscription_invitations_table'
```
Comment (gozineb) [inline scripts/tables.sql:175]: Modify last migration reference to yours

**Review by gozineb — CHANGES_REQUESTED** [review]
tables.sql to update

**Comment by mamadoudicko** [comment]
Agreed. We'll work on finding a way to automate this.

File: backend/utils/vectors.py (hunk 1 above)
Comment (mattzcarey) [reply backend/utils/vectors.py:70]: yep its a ulid now. We are handling them as strings I think

File: scripts/tables.sql (hunk 3 above)
Comment (mattzcarey) [reply scripts/tables.sql]: Done

**Review by gozineb — APPROVED** [review]

**Pull request merged**